posts/2024-10-08-24.0.0.10.adoc

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Support for Java SE 23 and new versionless MicroProfile features in 24.0.0.10"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dmuelle
+author_github: https://github.com/dmuelle
+seo-title: Support for Java SE 23 and new versionless MicroProfile features in 24.0.0.10 - OpenLiberty.io
+seo-description: This release introduces support for Java SE 23, which includes new features and enhancements. Versionless support for stand-alone MicroProfile features is also included.
+blog_description: This release introduces support for Java SE 23, which includes new features and enhancements. Versionless support for stand-alone MicroProfile features is also included.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Support for Java SE 23 and new versionless MicroProfile features in 24.0.0.10
+David Mueller <https://github.com/dmuelle>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+This release introduces support for Java SE 23, which includes new features and enhancements. Versionless support for stand-alone MicroProfile features is also included.
+
+
+In link:{url-about}[Open Liberty] 24.0.0.10:
+
+* <<#java, Support for Java 23 in Open Liberty>>
+* <<#mp, Versionless stand-alone MicroProfile Liberty features>>
+
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A240010+label%3A%22release+bug%22[24.0.0.10].
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
```

Comment:
Following this link shows several 2024 GA release blogs missing.  Those GA blog posts appears to not have any tags at all.

Reply:
Only affects draft branch.  Tag updates were probably not pushed back to that branch.  Mark resolved as such.